scripts/fobnail-mount.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description="Mount Fobnail-protected disk"
+After=network.target
```

Comment:
Right, it will help on boot, but IIUC this won't make service wait until `fobnail.link` and `fobnail.network` specifically are applied, so there is still a race condition between starting those scripts and assigning IP.

I don't know what happened to my other question, probably forgot to send it, but can we use `/lib/systemd/systemd-networkd-wait-online -i fobnail` in `mount.sh` or `ExecStartPre=`? It should help with this race condition, but I don't know if it will work with all different timeouts involved.

Reply:
Yes, waiting for network.target will delay start until basic network services are running, but at this point Fobnail won't have IP address assigned, I added `systemd-networkd-wait-online` to `ExecStartPre`, it does work (it does wait for the interface) but if Fobnail got disconnected before interface was configured it will hang in activating state, the service won't be stopped by udev because at that time it hasn't been started yet. It configured timeout to 5 sec to workaround this.